### ISSUE TITLE
Search: no a11y feedback for search results and search progress

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#2644]
* when no results are found, we need to indicate this
* when search is done we should indicate this with the number of results

Can try to use `role="status"` for this purpose.

**Comment by bpasero** [comment]
Pushed a change to set an element with the status role when search is finished. We have to verify that screen readers support the status role, otherwise I fear we would have to use the alert role for it which is a much stronger feedback that gets read immediately.

**Comment by isidorn** [comment]
Using VoiceOver I do not get any feedback when the search is finished, or when the search has no results. Moving to march since it is not critical.

**Comment by bpasero** [comment]
Looks like VoiceOver does not support role="status". My fix is to now always use role="alert" if on Mac and that seems to work.

**Comment by isidorn** [comment]
Oh yes, forgot to mention to you that I also had to use alert for my debug messages because status was not working

**Comment by bpasero** [comment]
@isidorn you can go back to using "status" now as I will take care of it by checking for the platform. Or are you saying that it even does not work on Windows?

**Comment by isidorn** [comment]
@bpasero no, I only checked on Mac. I will go back to using 'status'

**Comment by bpasero** [comment]
:+1:
